config/nightwatch.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   custom_commands_path: './test/util/nightwatch-commands',
   live_output: true,
   parallel_process_delay: 10,
-  disable_colors: false,
+  disable_colors: true,
```

Comment:
I'm fond of it locally because it makes for a nice visual breakup when we have _so much_ output. It's just a pain on Jenkins because it doesn't work.